core/templates/domain/exploration/exploration-permissions-backend-api.service.spec.ts

```diff
@@ -60,7 +57,8 @@ describe('Exploration permissions backend api service', () => {
       can_edit: true
     };
 
-    let expectedResponse = epof.createFromBackendDict(backendResponse);
+    let expectedResponse =
+      ExplorationPermissions.createFromBackendDict(backendResponse);
```

Comment:
```suggestion
    let expectedResponse = ExplorationPermissions.createFromBackendDict(
      backendResponse);
```

OR 

```
    let expectedResponse = (
      ExplorationPermissions.createFromBackendDict(backendResponse));
```

In general, break lines after the parenthesis.

Reply:
done!